skiko/src/iosMain/kotlin/org/jetbrains/skiko/Dispatchers.kt

```diff
@@ -14,8 +16,13 @@ import kotlin.native.internal.NativePtr
 // This is the only dispatcher that shall be used in Skiko on iOS.
 // Current (as of 1.5.2) dispatchers in kotlinx.coroutines are not usable
 // for needs of Skiko.
-@SharedImmutable
-internal val AppDispatcher: CoroutineDispatcher = NsQueueDispatcher(dispatch_get_main_queue())
+object SkikoDispatchers {
```

Comment:
If we intend to make it public, it is worth to describe why we don't use `Dispatchers.Main` or `Dispatchers.IO`. If it is a bug, maybe we should fill an issue in kotlinx.coroutines, and switch to `Dispatchers` after it will be fixed?

Also, if fixing `Dispatcher.Main` seems reasonable, because it a very specific dispatcher, tied to the platform, fixing `Dispatchers.IO` seems strange.

Reply:
Guess it’s a long story, and most likely we’ll need to expose that dispatcher at least to Compose.